about-page.css

```diff
@@ -0,0 +1,795 @@
+  .partner-mobile {
+    display: none;
+  }
+
+  /* nav banner starts */
+  .nav-banner {
+    background: #3e3e3e;
+  }
+
+  .nav-banner-list {
+    display: flex;
+    justify-content: end;
+    padding-right: 13%;
+    margin-block-end: 0;
+    padding-bottom: 0.42rem;
+  }
+
+  .nav-banner-item {
+    list-style-type: none;
+    color: #fff;
+    padding-left: 1.26rem;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 0.79rem;
+  }
+
+  /* nav banner ends */
+
+  /* navbar starts */
+  .hambar {
+    display: none;
+  }
+
+  .my-navbar-intro {
+    box-shadow: 0 15px 10px -15px #111;
+    padding-top: 13px;
+  }
+
+  .navbar {
+    padding-top: 0.84rem;
+  }
+
+  #no-wrap {
+    flex-wrap: nowrap;
+  }
+
+  .navbar-brand {
+    padding-top: 0;
+    padding-left: 9%;
+  }
+
+  .navbar-brand img {
+    max-width: 60%;
+    padding-bottom: 13px;
+  }
+
+  .navbar-ul {
+    display: flex;
+  }
+
+  .navbar-item {
+    padding-right: 3rem;
+    padding-top: 1.6%;
+    list-style: none;
+  }
+
+  .navbar-link {
+    color: #868686;
+    font-weight: 800;
+    font-size: 0.96rem;
+    text-decoration: none;
+  }
+
+  .navbar-content {
+    display: flex;
+
+    /* flex-wrap: inherit; */
+    align-items: center;
+
+    /* flex-direction: row-reverse; */
+    justify-content: space-between;
+    padding: 0;
+  }
+
+  .navbar-ALD {
+    border: 5px solid #ec4327;
+    color: #ec4327;
+    font-weight: 800;
+    font-size: 0.96rem;
+    text-decoration: none;
+    padding: 15px;
+  }
+
+  .first-navbar-wrapper {
+    width: 125%;
+    padding-bottom: 10px;
+  }
+
+  /* navbar ends */
+
+  /* Summit Description Starts */
+  .summit-description {
+    background-image:
+      linear-gradient(
+        rgba(255, 255, 255, 0.6),
+        rgba(255, 255, 255, 0.6)
+      ),
+      url(./new-img/about-img.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position-x: 50%;
+    margin-top: 0.2%;
+  }
+
+  .heading-wrapper {
+    display: none;
+  }
+
+  .summitjoin-wrapper {
+    text-align: center;
+    padding-left: 15%;
+    padding-right: 15%;
+    padding-top: 7%;
+  }
+
+  .summitdescription-join {
+    color: #ec5242;
+    font-weight: 700;
+    font-style: italic;
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  .summitjoin-border {
+    width: 4rem;
+    border-bottom: 3px solid #ec5242;
+    margin: 0 auto 1rem;
+    padding-top: 2%;
+  }
+
+  .summitdescription-details {
+    background: rgb(254, 250, 224, 0.4);
+    border: 1.5rem solid rgb(255, 255, 255, 0.4);
+    color: #000;
+    font-weight: 400;
+    font-size: 0.875rem;
+    text-align: center;
+    line-height: 25px;
+    margin-left: 7%;
+    margin-right: 7%;
+    display: none;
+  }
+
+  .summitdescription-details-desktop {
+    background: rgba(255, 255, 255, 0.7);
+    border: 65px solid rgba(255, 255, 255, 0.6);
+    color: #000;
+    text-align: center;
+    margin-left: 7%;
+    margin-right: 7%;
+    margin-top: 8%;
+  }
```

Comment:
- [ ] This is a continuation of the previous comment, please make sure to remove the transparency from the `background` property at line `568` as the Behance design is a solid white color. ✔